packages/ui5-middleware-cfdestination/README.md

```diff
@@ -34,6 +34,16 @@ whether to equip routes with authentication
 allow static assets to be picked up by the included `approuter`  
 defaults to `false` as usually all local files/assets are supposedly served by `ui5-server`
 
+- `rewriteContent`: `<boolean>`, default: `true`  
+enables/disables rewriting of the content by replacing
+the proxied url in the content with the server url
```

Comment:
ˋˋˋsuggestion
the proxied url in the response body with the server url
ˋˋˋ

Reply:
thx